yarn.lock

```diff
@@ -2651,7 +2663,21 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@jest/globals@^28.0.0-alpha.8, @jest/globals@workspace:*, @jest/globals@workspace:packages/jest-globals":
+"@jest/fake-timers@npm:^27.5.1":
```

Comment:
we're now pulling in more 27 stuff, can these remain `workspace:*`?

Reply:
probably from the one example using `workspace`...

Reply:
good catch.